src/parser/core/modules/Gauge/index.ts

```diff
@@ -1,3 +1,4 @@
 export {Gauge} from './Gauge'
 export {CounterGauge} from './CounterGauge'
+export type {CounterGaugeModifier} from './CounterGauge'
```

Comment:
Does this need the type specifier?

Reply:
Yes, our typescript settings make this required.